pipelines/internal/commands/run/run.go

```diff
@@ -587,54 +597,67 @@ func namedListOf(input, defaultPrefix string) map[string]string {
 	return output
 }
 
-func expandZones(project string, input []string) ([]string, error) {
-	var zones, prefixes []string
-	for _, zone := range input {
-		if strings.HasSuffix(zone, "*") {
-			prefixes = append(prefixes, zone[:len(zone)-1])
+func expandPrefixes(
```

Comment:
Don't break lines like this.  I don't imagine gofmt would even allow this.  Make sure you run gofmt.

Reply:
Didn't know about gofmt, but after running it, it allows the break lines. Nevertheless, if line length is not an issue, removed the breaks.